realm/realm-library/build.gradle

```diff
@@ -46,6 +46,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    compile "com.android.support:recyclerview-v7:23.1.1"
```

Comment:
This should just be provided. We don't want to force people to include RecyclerView if they don't use it.